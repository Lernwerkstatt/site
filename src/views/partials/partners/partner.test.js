const partner = require("./partner.hbs");

describe("partner.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      partner({
        img: "img/logo.png",
        name: "Partner X"
      })
    ).toMatchSnapshot();
  });
});
