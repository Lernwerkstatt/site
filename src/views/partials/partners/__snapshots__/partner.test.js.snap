// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`partner.hbs should match the snapshot 1`] = `
"<div class=\\"col-sm-4 align-self-center\\">
  <img src=\\"img/logo.png\\" class=\\"partner-img\\" alt=\\"Partner X\\" />
</div>
"
`;
